### PR TITLE
fixing anchor links

### LINKED ISSUE #4727: Broken links + anchors reported by the Link Checker
The broken links and anchors can be found in this deployment: https://vercel.com/prisma/docs/AgAsLWioZtDZf5jpi6m9Uue7FxsS#L113

### PR DESCRIPTION
**Pull request by meletj**
Fixing the broken links, broken anchor and domain URLs reported by link checker 
 
Fixes: https://github.com/prisma/docs/issues/4727

### CODE REVIEW

File: content/200-concepts/100-components/01-prisma-schema/03-generators.mdx [hunk 1]
```diff
@@ -57,7 +57,7 @@ In that case, Prisma detects your operating system and finds the right binary fi
 If you use macOS Intel x86 (`darwin`), then the binary file that was compiled for `darwin` will be selected.
 If you use macOS ARM64 (`darwin-arm64`), then the binary file that was compiled for `darwin-arm64` will be selected.
 
-> **Note**: The `native` binary target is the default. You can set it explicitly if you wish to [include additional binary targets for deployment to different environments](/guides/deployment/deployment-guides/deploying-to-aws-lambda#binary-targets-in-schemaprisma).
+> **Note**: The `native` binary target is the default. You can set it explicitly if you wish to [include additional binary targets for deployment to different environments](/guides/deployment/deployment-guides/use-prisma-with-serverless-framework#reduce-the-package-footprint).
```
Comment (jharrell) [inline content/200-concepts/100-components/01-prisma-schema/03-generators.mdx]:
```suggestion
> **Note**: The `native` binary target is the default. You can set it explicitly if you wish to [include additional binary targets for deployment to different environments](/guides/deployment/use-prisma-with-serverless-framework#binary-targets-in-schemaprisma).
```

**Review by jharrell — APPROVED** [review]

**Comment by nilubava** [comment]
@jharrell Could you please also fix the other broken links and anchors reported in the build log? https://vercel.com/prisma/docs/87sc9Vjjsd4U1WznR3JQMJp3CaAm#L82

**Comment by jharrell** [comment]
@nilubava happy to. Should we merge this PR first and then get an updated build log?

**Comment by meletj** [comment]
@jharrell The log in the current build should actually display updated broken links. So you should be able to add a commit to this PR to fix the deployment guides links..

**Comment by jharrell** [comment]
@nilubava @meletj things look good 

https://vercel.com/prisma/docs/3w5xPmHnJXAe9DArpqTpZwVXohHg#L82

Let me know if this looks good to you and I'll merge

**Comment by meletj** [comment]
> @nilubava @meletj things look good
> 
> https://vercel.com/prisma/docs/3w5xPmHnJXAe9DArpqTpZwVXohHg#L82
> 
> Let me know if this looks good to you and I'll merge

Looks great, thank Jon. Pls go ahead and merge.

**Pull request merged**